udata/core/spam/api.py

```diff
@@ -53,6 +53,5 @@ def get(self):
         discussions = Discussion.objects(Q(spam__status=POTENTIAL_SPAM) | Q(discussion__spam__status=POTENTIAL_SPAM))
 
         return [{
-            'title': discussion.spam_report_title(),
-            'link': discussion.spam_report_link(),
+            'message': discussion.spam_report_message([discussion]),
```

Comment:
Since we have only one model right now, we could have more information… I think we can wait to have multiple models to see if we can find a better response here.